docs/sdkx-cocos2dx/getting-started.mdx

```diff
@@ -0,0 +1,308 @@
+
+#### Along with `NSPhotoLibraryUsageDescription` (as mentioned above), also add `NSCameraUsageDescription` and `NSMicrophoneUsageDescription` key in your application's info.plist file
+
+If your game does not use this permission, you would need to add this key as well as description for the same. Not adding this key-description pair might cause game rejection.
+
+**NSCameraUsageDescription Description text:** "We need to access the camera to allow users to click images meant to be sent as an attachment for help and support reasons."
+
+**NSMicrophoneUsageDescription Description text:** "We need to access the microphone to allow users to record and send videos meant to be sent as an attachment for help and support reasons."
+
+Note that this is just a suggested description. If you need localisations for the same, please <a href="https://support.helpshift.com/kb/contact/" target="_blank">Contact Us</a>
+
+</Admonition>
+
+<Admonition type="caution" title="Warning">
+
+Attachments uploads are not supported on iOS 10.x and below due to issues with permissions.
+
+</Admonition>
+
+</TabItem>
+
+</Tabs>
+
+## Step 3 - Initialize Helpshift in your game {#initialize}
+
+1. First, create a game on the Helpshift Dashboard
+
+   <Image src="/static/books/helpshiftx/add_app.png" width="full" />{" "}
+
+2. Create a game with **Android** and **iOS** as a selected `Platform`
+   <Image
+     src="/static/books/helpshiftx/create_app_with_android_platform.png"
+     width="full"
+   />{" "}
+
+Helpshift uniquely identifies each registered game with a combination of 2 tokens:
+
+<div className="compact">
+
+|               |                                                                                |
+| ------------- | ------------------------------------------------------------------------------ |
+| `Platform ID` | Your Games's unique App Id (Games's `App Id` on dashboard is your platform Id) |
+| `Domain Name` | Your Helpshift domain. E.g. _happyapps.helpshift.com_                          |
+
+</div>
+
+To obtain these values:
+
+1. Go to your Helpshift dashboard
+2. Navigate to `Settings` > `SDKs (for Developers)`
+3. Select your Game and from the dropdowns
+4. Copy the two tokens present in the "Android" table
+
+<Image src="/static/books/helpshiftx/copy_install_keys.png" width="full" />
+
+<Tabs groupId="platform">
+
+<TabItem value="android" label="Android">
+
+Initialize Helpshift by importing `com.helpshift.HelpshiftCocosBridge` and
+calling the `HelpshiftCocosBridge.install` in the `onCreate` of your `Cocos2dxActivity`:
+
+```java
+import com.helpshift.HelpshiftCocosBridge;
+
+public class MyActivity extends Cocos2dxActivity {
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        // Add install configs in the config map
+        Map<String, Object> config = new HashMap<>();
+        //...
+        // Install call
+        HelpshiftCocosBridge.install(
```

Comment:
Are we not suggesting install call from C++ layer for android ?

Reply:
No, In android `install` call can only be done from java layer as we expect an activity context